IdentityCore/src/IdentityCore_Internal.h

```diff
@@ -31,6 +31,7 @@
 #import "MSIDLogger+Internal.h"
 #import "MSIDError.h"
 #import "MSIDOAuth2Constants.h"
+#import "MSIDCredentialCacheItem+MSIDBaseToken.h"
```

Comment:
I don't think this needs to be in the Internal header. It's probably only used in 2-3 classes.

Reply:
It is also used in ADAL, so if we move it out from the internal header, we need to make sure we update ADAL too.

Reply:
I just checked-in this change, should I back it out?

Reply:
The internal file imports headers that are used almost everywhere. MSIDCredentialCacheItem doesn't qualify for it. As a rule, when something is merged in common, MSAL and ADAL need to be updated to the latest common core to make sure nothing breaks anyway.